notebooks/003-hello-segmentation/003-hello-segmentation.ipynb

```diff
@@ -69,17 +68,17 @@
    "cell_type": "code",
    "execution_count": null,
    "source": [
-    "# The segmentation network expects images in RGB format\n",
-    "image = cv2.cvtColor(cv2.imread(\"data/coco_cross.png\"), cv2.COLOR_BGR2RGB)\n",
+    "# The segmentation network expects images in BGR format\n",
+    "image = cv2.cvtColor(cv2.imread(\"data/empty_road_mapillary.jpg\"), cv2.COLOR_RGB2BGR)\n",
```

Comment:
```suggestion
    "image = cv2.imread(\"data/empty_road_mapillary.jpg\")\n",
```

`cv2.imread` by default loads the image as BGR, which is expected by the model. But with that, visualization will look off, because matplotlib expects RGB, so then you can created an `rgb_image` with `cv2.cvtColor(image, cv2.COLOR_BGR2RGB)` for visualization. See https://python.plainenglish.io/opencv-python-tutorial-for-beginners-part-1-7639496e98a

Reply:
Thanks Helena. I was wondering why the color was off.

Reply:
Added one more variable that holds RGB picture as an example how to convert it back to format that can be displayed using matplotlib. Hope that is fine